src/androidTest/java/com/mixpanel/android/mpmetrics/MixpanelNotificationBuilderTest.java

```diff
@@ -200,6 +248,116 @@ public void testNoActionButtons() {
         verify(builderSpy, never()).addAction(any(Notification.Action.class));
     }
 
+    public void testValidNotificationBadge() {
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        intent.putExtra("mp_bdgcnt", 2);
+        mpPushSpy.createNotification(intent);
+        verify(builderSpy).setNumber(2);
+    }
+
+    public void testInvalidNotificationBadge() {
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        intent.putExtra("mp_bdgcnt", "0");
+        mpPushSpy.createNotification(intent);
+        verify(builderSpy, never()).setNumber(any(Integer.class));
+    }
+
+    public void testChannelId() {
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        intent.putExtra("mp_channel_id", "12345");
+        Notification notification = mpPushSpy.createNotification(intent);
+        verify(builderSpy).setChannelId("12345");
+        assertEquals(notification.getChannelId(), "12345");
+    }
+
+    public void testNoChannelId() {
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        Notification notification = mpPushSpy.createNotification(intent);
+        verify(builderSpy).setChannelId(anyString());
```

Comment:
can't we be more specific here and verify that `setChannelId(MixpanelPushNotification.NotificationData.DEFAULT_CHANNEL_ID)`?